hat/app/org/hatdex/hat/resourceManagement/HatServerProvider.scala

```diff
@@ -82,39 +82,40 @@ class HatServerProviderImpl @Inject() (
     configuration.get[FiniteDuration]("resourceManagement.serverIdleTimeout")
 
   def retrieve(hatAddress: String): Future[Option[HatServer]] = {
+    logger.warn(s"HatServiceProvider.retrieve: looking up hatAddress: server:${hatAddress}")
+
     cache
       .get[HatServer](s"server:$hatAddress")
       .flatMap {
         case Some(server) => Future.successful(Some(server))
         case _ =>
           (serverProviderActor ? HatServerProviderActor.HatServerRetrieve(
-            hatAddress
-          )) map {
-            case server: HatServer =>
-              logger.debug(s"Got back server $server")
-              cache.set(s"server:$hatAddress", server, serverInfoTimeout)
-              Some(server)
-            case error: HatServerDiscoveryException =>
-              logger.warn(s"Got back error $error")
-              throw error
-            case message =>
-              logger.warn(
-                s"Unknown message $message from HAT Server provider actor"
-              )
-              val error = new HatServerDiscoveryException("Unknown message")
-              throw error
-          } recoverWith {
-            case e =>
-              logger.warn(
-                s"Error while retrieving HAT $hatAddress info: ${e.getMessage}"
-              )
-              val error = new HatServerDiscoveryException(
-                "HAT Server info retrieval failed",
-                e
-              )
-              throw error
-          }
+                hatAddress
```

Comment:
did you run scalafmt ? im seeing a tonne of whitespace changes

Reply:
scalafmt is in my pre-commit.